src/mechanics/timeIntegration/TimeIntegrationBase.h

```diff
@@ -238,10 +189,6 @@ class TimeIntegrationBase
         mCallback = rCallback;
     }
 
-    int GetIterationCount() const
-    {
-        return mIterationCount;
```

Comment:
I think I use this method in my main files from time to time. If so, I will commit a test soon.